src/docs/src/translation/id.json

```
@@ -0,0 +1,329 @@
+  "Rating is a set of radio buttons that allow the user to rate something.": "Rating is a set of radio buttons that allow the user to rate something.",
+  "Select is used to pick a value from a list of options.": "Select is used to pick a value from a list of options.",
+  "Stack visually puts elements on top of each other.": "Stack visually puts elements on top of each other.",
+  "Stat is used to show numbers and data in a box.": "Stat is used to show numbers and data in a box.",
+  "Steps can be used to show a list of steps in a process.": "Steps can be used to show a list of steps in a process.",
+  "Swap allows you to toggle the visibility of two elements using a checkbox or a class name.": "Swap allows you to toggle the visibility of two elements using a checkbox or a class name.",
+  "Tabs can be used to show a list of links in a tabbed format.": "Tabs can be used to show a list of links in a tabbed format.",
+  "Table can be used to show a list of data in a table format.": "Table can be used to show a list of data in a table format.",
+  "Textarea allows users to enter text in multiple lines.": "Textarea allows users to enter text in multiple lines.",
+  "Toggle is a checkbox that is styled to look like a switch button.": "Toggle is a checkbox that is styled to look like a switch button.",
+  "Tooltip can be used to show a message when hovering over an element.": "Tooltip can be used to show a message when hovering over an element.",
+  "Code mockup": "Code mockup",
+  "Phone mockup": "Phone mockup",
+  "Window mockup": "Window mockup",
+  "Text Input": "Text Input",
+  "Range slider": "Range slider",
+  "Tabs": "Tabs",
+  "All daisyUI components": "All daisyUI components",
+  "Class name": "Class name",
+  "Type": "Type",
+  "Search": "Search",
+  "daisyUI themes": "daisyUI themes",
+  "How to use daisyUI themes?": "Bagaimana cara menggunakan tema daisyUI?",
+  "daisyUI Theme Generator": "Generator tema daisyUI",
+  "You can add your custom themes to <span class='badge badge-outline'>tailwind.config.js</span> file in <span class='badge badge-outline'>daisyui > themes</span> array. In this page, you can pick required color values and see how the components will look like with them.": "Anda dapat menambahkan tema kustom Anda ke dalam file <span class='badge badge-outline'>tailwind.config.js</span> di array <span class='badge badge-outline'>daisyui > themes</span>. Pada halaman ini, Anda akan memilih nilai warna yang dibutuhkan dan melihat bagaimana komponen akan terlihat ketika warna diterapkan.",
+  "You can also define optional colors to have more control on the color values (for example: the color of button when it's focused of the color of the text on a button)": "Anda juga bisa mendifinisikan warna opsional untuk mendapatkan kontrol lebih terhadap nilai warna (contohnya: warna teks dari tombol yang terfokus).",
```

tbh I'm not really sure what `the color of button when it's focused of the color of the text on a button` means

```suggestion
  "You can also define optional colors to have more control on the color values (for example: the color of button when it's focused of the color of the text on a button)": "Anda juga bisa mendefinisikan warna opsional untuk mendapatkan kontrol lebih terhadap nilai warna (contoh: warna teks dari tombol yang sedang dalam fokus).",
```